nbconvert/preprocessors/tests/test_execute.py

```diff
@@ -195,19 +197,37 @@ def timeout_func(source):
 
     def test_allow_errors(self):
         """
-        Check that conversion continues if ``allow_errors`` is False.
+        Check that conversion halts if ``allow_errors`` is False.
         """
         current_dir = os.path.dirname(__file__)
         filename = os.path.join(current_dir, 'files', 'Skip Exceptions.ipynb')
         res = self.build_resources()
         res['metadata']['path'] = os.path.dirname(filename)
         with pytest.raises(CellExecutionError) as exc:
             self.run_notebook(filename, dict(allow_errors=False), res)
-        self.assertIsInstance(str(exc.value), str)
-        if sys.version_info >= (3, 0):
-            assert u"# üñîçø∂é" in str(exc.value)
-        else:
-            assert u"# üñîçø∂é".encode('utf8', 'replace') in str(exc.value)
+            self.assertIsInstance(str(exc.value), str)
+            if sys.version_info >= (3, 0):
+                assert u"# üñîçø∂é" in str(exc.value)
+            else:
+                assert u"# üñîçø∂é".encode('utf8', 'replace') in str(exc.value)
+
+    def test_raises_exception_cell_tag(self):
+        """
+        Check that conversion continues if ``raises-exception`` is present in
+        the cell in which an exception is raised.
```

Comment:
This one is actually testing `force_raise_errors`, though.

Can we split this into two tests: one with this name and docstring, and another one with this code (`test_force_raise_errors`)? ;-)

Reply:
Technically the test of it on its own implicitly occurs earlier in the global "convert everything!"  test: https://github.com/jupyter/nbconvert/blob/0a262a3a00a1680c0b30c95b0b9837ed4f8ff063/nbconvert/preprocessors/tests/test_execute.py#L122-L137

But you're right that this should explicitly test `force_raise_errors` instead

Reply:
Aha,  great. In that case, if you just want to fix the test name and docstring,  I think this is ready to go.